src/psyclone/tests/test_files/dynamo0p3/testkern_write_w2broken_w1_mod.f90

```diff
@@ -47,15 +47,19 @@ module testkern_write_w2broken_w1_mod
 
   type, extends(kernel_type) :: testkern_write_w2broken_w1_type
      private
-     type(arg_type), dimension(8) :: meta_args = (/   &
-          arg_type(gh_field,   gh_read,  w3),         &
-          arg_type(gh_field,   gh_read,  w0),         &
-          arg_type(gh_field,   gh_read,  w2),         &
-          arg_type(gh_field,   gh_inc,   w1),         &
-          arg_type(gh_field*3, gh_read,  wchi),       &
-          arg_type(gh_field,   gh_read,  wtheta),     &
-          arg_type(gh_field,   gh_read,  w2h),        &
-          arg_type(gh_field,   gh_write, w2broken)    &
+     type(arg_type), dimension(12) :: meta_args = (/ &
+          arg_type(gh_field,   gh_read,  w3),        &
+          arg_type(gh_field,   gh_read,  w0),        &
+          arg_type(gh_field,   gh_read,  w2),        &
+          arg_type(gh_field,   gh_inc,   w1),        &
+          arg_type(gh_field*3, gh_read,  wchi),      &
+          arg_type(gh_field,   gh_read,  wtheta),    &
+          arg_type(gh_field,   gh_read,  w2h),       &
+          arg_type(gh_field,   gh_write, w2broken),  &
```

Comment:
Is there a particular reason why you've modified this test kernel?

Reply:
The description in `1.5.1_single_invoke_write_multi_fs.f90` says "invoke call using all function spaces". Also, it is practical to check the continuity of `W2Vtrace` in `test_arg_discontinuous`.

Reply:
OK, thanks.